lighthouse-core/gather/gatherers/meta-elements.js

```diff
@@ -25,6 +25,7 @@ class MetaElements extends Gatherer {
         return {
           name: meta.name.toLowerCase(),
           content: meta.content,
+          property: meta.attributes.property ? meta.attributes.property.value : '',
```

Comment:
this should be `undefined` instead of `''` when not found

Reply:
I was in doubt about this, because `content` is `''`, also if it's not defined. So I kept it in line. But since `property` isn't in the HTML5 spec, I understand your comment as well.